hack/release

```diff
@@ -20,6 +21,14 @@ if [ -n "$CACHE_FROM" ]; then
 fi
 if [ -n "$CACHE_TO" ]; then
   for cto in $CACHE_TO; do
+    if [[ $cto == *"type=gha"* ]]; then
+      if [[ -n "$GITHUB_REPOSITORY" ]] && [[ $cto != *"repository="* ]]; then
+        cto="${cto},repository=${GITHUB_REPOSITORY}"
+      fi
+      if [[ -n "$GITHUB_TOKEN" ]] && [[ $cto != *"ghtoken="* ]]; then
+        cto="${cto},ghtoken=${GITHUB_TOKEN}"
+      fi
```

Comment:
@tonistiigi These attributes should only be set for `cache-to` when making the batch request to list cache keys but not `cache-from` right?

Reply:
I think it is better to be consistent here and set both. Atm it would be only used for export path but I think this should be internal behavior that may change in the future.